### PR TITLE
WEB-10426

### PR DESCRIPTION
**Pull request by matiaslg13**
- Updating SDK after new/updated endpoints

### CODE REVIEW

**Review by Krithik-Serato — APPROVED** [review]

**Review by AnnieCSerato — APPROVED** [review]
Nice one, looks good to me.
Please make sure you install and test this in the client app thats using it before making a proper release.

**Comment by matiaslg13** [comment]
Thank you @AnnieCSerato @Krithik-Serato !

File: src/Identity.js [hunk 1]
```diff
@@ -222,4 +222,38 @@ export default class IdentityService extends Service {
...
+   * Updates user's details (email address)
+   *
+   * @param {Object} param Options
+   * @param {String} param.emailAddress Email address to change to
+   * @returns {Promise<User>}
+   * **/
+  updateUser ({ emailAddress }) {
+    return this.fetch(
+      this.bearerTokenAuthHeader(),
+      this.userId === 0 ? '/api/v1/me' : '/api/v1/users/' + this.userId,
```
Comment (mshahidi) [inline src/Identity.js:236]: Isn't this only `/users` endpoint? I don't think we have `/me` endpoint for it, right?

Comment (matiaslg13) [reply src/Identity.js]: You're right, @mshahidi . We only have this working for `/users` at this stage. Thanks! I've just updated it.

**Pull request merged**